client/src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { fab} from '@fortawesome/free-brands-svg-icons';
+import {faHeart, faCompass, faSearch, faUser} from '@fortawesome/free-solid-svg-icons';
+import '../styles/Menu.scss';
+
+library.add(fab, faHeart, faCompass, faSearch, faUser);
+class Menu extends React.Component {
+    public render():
+    JSX.Element {
+        return (
+      <nav className='navig'>
+        <div className='logo_component'>
+        <FontAwesomeIcon className='inst_icon' icon={['fab', 'instagram']} size='lg'/>
+        <h1 className='jsgram'>JSGram</h1>
+        </div>
+        <div className='center-component'>
+          <FontAwesomeIcon className='find_icon' icon={faSearch} size='xs'/>
+        <input type='search' className='input_input' name='q' placeholder='Search'></input>
```

Comment:
is this custom class or bootstrap?

Reply:
it is a custom class. I do not use bootstrap in menu

Reply:
then it is a bad name for class. Also pls use one convension the same as others, I mean '-' or '_'

Reply:
the same for name